src/core/constants.js

```diff
@@ -195,8 +196,17 @@ export const INSTALL_ERROR = 'INSTALL_ERROR';
 
 // Tracking categories.
 export const INSTALL_CATEGORY = 'AMO Addon / Theme Installs';
+export const INSTALL_THEME_CATEGORY = 'AMO Theme Installs';
+export const INSTALL_EXTENSION_CATEGORY = 'AMO Addon Installs';
+
 export const INSTALL_STARTED_CATEGORY = 'AMO Addon / Theme Installs Started';
```

Comment:
Do we still need this? Won't it always be either `INSTALL_THEME_STARTED_CATEGORY` or `INSTALL_EXT_STARTED_CATEGORY`?

Also, for consistency it might be better to make the latter the full `INSTALL_EXTENSION_STARTED_CATEGORY`.

Reply:
this was set up this way so the "general" category could be passed in and then depending on the type, the more descriptive category was used. I'll take another look tho to see about cleaning this up.

yes, I meant to change ext => extension. Thanks for catching that one :)